### DISCUSSION TITLE
display pgn comments

### DISCUSSION THREAD

**Discussion by arnaule** [rodpolako/Chess-PGN-Trainer#10] [Ideas]
When there is a comment on a move in the pgn, it would be nice to display it on the screen

**Comment by rodpolako** [comment]
Adding additional information from the PGN is on the roadmap and will be in a future release but is still in the design phase.  Which fields/comments were you thinking about?

**Comment by arnaule** [comment]
hello, Thank you. Any field between { or {[ is interesting : it is either personal annotations on the move, or computer eval of the position

**Reply by rodpolako** [reply]
Thanks for the suggestion.  I'm not sure about computer evaluation since that is not the purpose of the tool but I will need to think about personal annotations.  I would need to figure out a way to implement that doesn't add to visual clutter.  The design goal is for **testing/drilling** as efficiently as possible so I'm only looking at features that support that purpose.

For learning, you may want to check out creating a study on Lichess as that will allow you to import a PGN with annotation as well as much more while also giving you access to Stockfish for analysis.

**Comment by rodpolako** [comment]
Hi arnaule,
   All pgn annotations are now showing in the application.  This includes any written comments as well as any circles or arrows.  Check it out.
